### PR TITLE
Packs - added eula link

### PR DESCRIPTION
**Pull request by IkaDemisto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold

## Related Issues
related to: https://github.com/demisto/etc/issues/25157

## Description
Added `eulaLink` field to packs metadata.

### CODE REVIEW

**Comment by IkaDemisto** [comment]
@yuvalbenshalom please merge

File: Tests/Marketplace/marketplace_services.py [hunk 1]
```diff
@@ -405,6 +410,29 @@ def _get_author(support_type, author=None):
...
+
+        Returns:
+            str: certification value
+        """
+        if support_type == Metadata.XSOAR_SUPPORT:
+            return Metadata.CERTIFIED
+        elif support_type != Metadata.XSOAR_SUPPORT and certification:
+            return certification
+        else:
+            return ""
```
Comment (yuvalbenshalom) [inline Tests/Marketplace/marketplace_services.py:434]: @IkaDemisto 
are you sure the correct value here should be empty and not none?
in the server the values are `none`, `verified`, `certified`.
@asafshen WDYT?

Comment (IkaDemisto) [reply Tests/Marketplace/marketplace_services.py:434]: @yuvalbenshalom `none` or empty string will have same result.

Comment (yuvalbenshalom) [reply Tests/Marketplace/marketplace_services.py:434]: @IkaDemisto please send me a screen shot of the result

**Pull request merged**